docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -58,8 +58,10 @@ hs.refreshIndex("empIndex", "full")
 After some files are added to or deleted from the original source files, an index was built on,
 using `refreshIndex` with the `"incremental"` mode causes the index refresh action recreate any existing index file,
 which has some deleted records, to remove those records. Refresh also creates new index files by indexing newly added data files.
-An index needs to have lineage enabled to be eligible for refresh in the incremental mode.
-Check the [configuration](02-ug-configuration.md) page to see how lineage is enabled when creating an index.
+An index needs to have lineage enabled to support deletes during refresh in the incremental mode. Lineage 
```

Comment:
Can we create a sub section for lineage (with 6 of #?)? It's pre-requisite for delete refresh/hybrid scan, so I think it's better to make it noticeable.

Reply:
Sure, but where do you exactly want to put that section? I am thinking it should be with "3 #" and comes before "Options of using index when your dataset changes", as it is used by refresh/hybridScan which come in that section. @sezruby What do you think?

Reply:
How about "### Pre-requisite" before "### RefreshIndex"?

Reply:
I added it as "### Lineage"

Reply:
Thanks! and then now we can remove ... Lineage~ part here and, add link to the section. 

```suggestion
An index needs to have [lineage](#lineage) enabled to support deletes during refresh in the incremental mode. 
```